From 983e3532c2de15df70aaf80b8906be2d0734dc37 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rafa=C5=82=20Mi=C5=82ecki?= <zajec5@gmail.com>
Date: Fri, 5 Dec 2014 12:16:12 +0100
Subject: [PATCH] ARM: BCM5301X: Switch DT files to "gpio-keys-polled"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We don't support GPIO interrupts yet, so use this as a workaround.

Signed-off-by: Rafał Miłecki <zajec5@gmail.com>
---
 arch/arm/boot/dts/bcm4708-buffalo-wzr-1750dhp.dts  | 2 +-
 arch/arm/boot/dts/bcm4708-netgear-r6250.dts        | 2 +-
 arch/arm/boot/dts/bcm4708-netgear-r6300-v2.dts     | 2 +-
 arch/arm/boot/dts/bcm47081-asus-rt-n18u.dts        | 2 +-
 arch/arm/boot/dts/bcm47081-buffalo-wzr-600dhp2.dts | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

--- a/arch/arm/boot/dts/bcm4708-buffalo-wzr-1750dhp.dts
+++ b/arch/arm/boot/dts/bcm4708-buffalo-wzr-1750dhp.dts
@@ -92,9 +92,10 @@
 	};
 
 	gpio-keys {
-		compatible = "gpio-keys";
+		compatible = "gpio-keys-polled";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		poll-interval = <200>;
 
 		restart {
 			label = "Reset";
--- a/arch/arm/boot/dts/bcm4708-netgear-r6250.dts
+++ b/arch/arm/boot/dts/bcm4708-netgear-r6250.dts
@@ -69,9 +69,10 @@
 	};
 
 	gpio-keys {
-		compatible = "gpio-keys";
+		compatible = "gpio-keys-polled";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		poll-interval = <200>;
 
 		wps {
 			label = "WPS";
--- a/arch/arm/boot/dts/bcm4708-netgear-r6300-v2.dts
+++ b/arch/arm/boot/dts/bcm4708-netgear-r6300-v2.dts
@@ -59,9 +59,10 @@
 	};
 
 	gpio-keys {
-		compatible = "gpio-keys";
+		compatible = "gpio-keys-polled";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		poll-interval = <200>;
 
 		wps {
 			label = "WPS";
--- a/arch/arm/boot/dts/bcm47081-asus-rt-n18u.dts
+++ b/arch/arm/boot/dts/bcm47081-asus-rt-n18u.dts
@@ -59,9 +59,10 @@
 	};
 
 	gpio-keys {
-		compatible = "gpio-keys";
+		compatible = "gpio-keys-polled";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		poll-interval = <200>;
 
 		restart {
 			label = "Reset";
--- a/arch/arm/boot/dts/bcm47081-buffalo-wzr-600dhp2.dts
+++ b/arch/arm/boot/dts/bcm47081-buffalo-wzr-600dhp2.dts
@@ -92,9 +92,10 @@
 	};
 
 	gpio-keys {
-		compatible = "gpio-keys";
+		compatible = "gpio-keys-polled";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		poll-interval = <200>;
 
 		aoss {
 			label = "AOSS";
--- a/arch/arm/boot/dts/bcm4708-luxul-xwc-1000.dts
+++ b/arch/arm/boot/dts/bcm4708-luxul-xwc-1000.dts
@@ -47,9 +47,10 @@
 	};
 
 	gpio-keys {
-		compatible = "gpio-keys";
+		compatible = "gpio-keys-polled";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		poll-interval = <200>;
 
 		restart {
 			label = "Reset";
--- a/arch/arm/boot/dts/bcm47081-buffalo-wzr-900dhp.dts
+++ b/arch/arm/boot/dts/bcm47081-buffalo-wzr-900dhp.dts
@@ -25,9 +25,10 @@
 	};
 
 	gpio-keys {
-		compatible = "gpio-keys";
+		compatible = "gpio-keys-polled";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		poll-interval = <200>;
 
 		restart {
 			label = "Reset";
